hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/server/ProfileServlet.java

```diff
@@ -208,11 +208,11 @@ protected void doGet(final HttpServletRequest req,
       return;
     }
     // if pid is explicitly specified, use it else default to current process
-    pid = getInteger(req, "pid", pid);
+    Integer processId = getInteger(req, "pid", pid);
 
     // if pid is not specified in query param and if current process pid
```

Comment:
should we update the comment? It seems we are not trying to determine current process pid?

Reply:
It is determined in `getPid()` call from constructor:

https://github.com/apache/hadoop/blob/aa664d72595ddfcb1a1bf082381bb222e59db354/hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/server/ProfileServlet.java#L126-L128